### PR TITLE
VAGOV-5766: Changing node type meta tag order.

### PR DESCRIPTION
**Pull request by ethanteague**
## What this does
Sets metatags for these content types
- Press release
- News Story
- Event

To this pattern: [node:field_office] | [node:title] | [site:name]

### CODE REVIEW

File: config/sync/metatag.metatag_defaults.node__event.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@ dependencies: {  }
 id: node__event
 label: 'Content: Event'
 tags:
-  title: '[node:title] | [node:field_office] | [site:name]'
-  og_image_width: '600'
+  title: '[node:field_office] | [node:title] | [site:name]'
   og_image: '[node:field_media:entity:image:entity:url]'
+  og_image_width: '600'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__event.yml]: was this intentional?

Comment (kevwalsh) [reply config/sync/metatag.metatag_defaults.node__event.yml]: has this been tested for whether or not it messes with the aspect ratio? i didn't really expect og_image_height to be specified either, on the next line, but i worry about specifying both.

Comment (ethanteague) [reply config/sync/metatag.metatag_defaults.node__event.yml]: Updated and removed, @kevwalsh

File: config/sync/metatag.metatag_defaults.node__event.yml [hunk 2]
```diff
@@ -5,8 +5,7 @@ dependencies: {  }
 id: node__event
 label: 'Content: Event'
 tags:
-  title: '[node:title] | [node:field_office] | [site:name]'
-  og_image_width: '600'
```
Comment (swirtSJW) [inline config/sync/metatag.metatag_defaults.node__event.yml]: @ethanteague  Was this supposed to go away?  It seems unrelated to the ticket.

Comment (ethanteague) [reply config/sync/metatag.metatag_defaults.node__event.yml]: Yep. It was added in inadvertently.
